src/main/scala/DataBaseConn.scala

```diff
@@ -0,0 +1,38 @@
+//DataBaseConnection
+
+object DataBaseConn{
+  import cats.effect.IO
+  import doobie._
+  import doobie.implicits._
+
+  import scala.concurrent.ExecutionContext
+
+  implicit val cs = IO.contextShift(ExecutionContext.global)
+
+  val xa = Transactor.fromDriverManager[IO](
+    "org.postgresql.Driver", "jdbc:postgresql:browserhistory", "postgres", ""
+  )
```

Comment:
Since you added HikariCP (doobie-hikari) to the dependency list, you may wish to use it for better performance.
```
    private val dataSource = {
      val ds = new HikariDataSource
      ds.setDriverClassName(dbSettings.driver)
      ds.setJdbcUrl(dbSettings.url)
      ds.setUsername(dbSettings.user)
      ds.setPassword(dbSettings.password)
      ds
    }

    private val xa: Transactor[IO] = HikariTransactor[IO](
      dataSource,
      connectEC = connectEC,
      blocker = blocker
    )
```

Reply:
You can also check the docs here: https://tpolecat.github.io/doobie/docs/14-Managing-Connections.html